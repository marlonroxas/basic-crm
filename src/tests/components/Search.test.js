import React from 'react'
import { shallow } from 'enzyme'

import Search from '../../components/Search/index'

test('Should render Search correctly', () => {
  const wrapper = shallow(<Search />)
  expect(wrapper).toMatchSnapshot()
})