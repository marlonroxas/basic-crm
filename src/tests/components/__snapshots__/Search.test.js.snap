// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Search correctly 1`] = `
<form
  className="search-form"
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <input
      onChange={[Function]}
      placeholder="Search"
      type="text"
      value=""
    />
    <button
      className="primary-btn"
      type="submit"
    >
      Search
    </button>
  </div>
</form>
`;
